schemas/vulnerability.yml

```diff
@@ -0,0 +1,159 @@
+---
+- name: vulnerability
+  title: Vulnerability
+  group: 2
+  short: Fields to describe the vulnerabilty relevant to an event.
+  description: >
+    The vulnerability fields describe information about a vulnerabilty that is
+    relevant to an event.
```

Comment:
Typo first "vulnerabilty" => "vulnerability", then a more philosophical point:

The description of the field set makes it sound like users are expected to enrich "normal" events with vulnerability information. Is that our expectation?

My expectation would be more along the lines of users *directly importing scan findings*. In other words, this isn't another kind of event being enriched with this info. The event/document coming in is literally one finding from the report, which is getting imported for correlation in the SIEM.

If you agree with this, we should address this directly. Perhaps:

```YAML
  short: Fields to describe a vulnerability finding.
  description: >
    The vulnerability fields describe information about a vulnerability that was
    reported by a scan.
```

Reply:
Fixed typo.

I think adjusting this description makes sense. That said, just as some context: https://github.com/elastic/ecs/pull/581#issuecomment-555550726